.circleci/config.yml

```diff
@@ -195,9 +195,24 @@ jobs:
     steps:
       - checkout:
           path: ~/project
+
       # attach website build directory
       - attach_workspace:
           at: ~/project/website
+
+      # restores gem cache
+      - restore_cache:
+          key: *RUBYGEM_CACHE_KEY
+      # rerun build with 'ENV=production' to add analytics
+      - run:
+          name: install gems
+          command: bundle check || bundle install --path vendor/bundle --retry=3
+
+      # rerun build with 'ENV=production' to add analytics
+      - run:
+          name: middleman build
+          command: bundle exec middleman build
```

Comment:
Does this implicitly set `ENV=production`? I'm confused otherwise how that is being passed through.

Reply:
It gets it from the circleci `context`.

Reply:
The environment variable is set in the context but the actual ID that is based on the environment variable is here: https://github.com/hashicorp/consul/blob/master/website/config.rb#L15-L21